src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,32 @@
+import { DeleteOutlineSharp } from "@mui/icons-material";
+import { IconButton, Typography } from "@mui/material";
+import { DeleteCommentApi } from "../utils/api";
+
+export default function DeleteComment({ commentId, setComments }) {
+  function handleClick(event) {
+    event.preventDefault();
+    setComments((prevComments) => {
+      return prevComments.filter((comment) => {
+        return comment.comment_id !== commentId;
+      });
+    });
+
+    DeleteCommentApi(commentId).then(() => {});
```

Comment:
this function shouldn't start with a capital letter

Reply:
it would also be better to see that the function is deleted before completely removing off the screen